src/pages/404.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from "next/link";
+import { Flex, Button, Text } from "@chakra-ui/react";
+import UnSmile from "@/assets/svg/404.svg";
+import HOME_PAGE_LINK from "../constants";
```

Comment:
The home page link may not be https://courtcanva.com/. It may go to the UAT version home page.
Using a variable in .env file to define this maybe a better approach.

Reply:
it has been fixed , thanks